src/api/app/controllers/webui/user_controller.rb

```diff
@@ -149,11 +149,20 @@ def icon
   end
 
   def register
-    opts = { login:    params[:login],
-             email:    params[:email],
-             realname: params[:realname],
-             password: params[:password],
-             state:    params[:state] }
+    opts = { login:           params[:login],
+             email:           params[:email],
+             realname:        params[:realname],
+             password:        params[:password],
+             password_repeat: params[:password_repeat],
+             state:           params[:state] }
+
+    if params[:password] != params[:password_repeat]
```

Comment:
Why validations in a controller? I don't think this should be model related. The model should just accept a password otherwise we've to create users and provide a hash containing `password` and `password_repeat` just for creating users on the command line or via the API. I think this should be only validated for the sign up process on the webui. That's the reason why I didn't put it inside a model. Also a side-effect of this would be, that we would need to adjust the API which I guess we don't want.